src/main/java/co/rsk/federate/btcreleaseclient/BtcReleaseClient.java

```diff
@@ -551,13 +551,16 @@ public void onBtcRelease(BtcTransaction signedBtcTx) {
     This way the pegoutBtcTx has the same hash as the one registered in release_requested event topics.
      */
     protected void removeSignaturesFromTransaction(BtcTransaction pegoutBtcTx, Federation spendingFed) {
+        if (pegoutBtcTx.hasWitness()) {
+            return;
```

Comment:
Do you think we should log something when the tx has witness?

Reply:
something like

`logger.debug("[removeSignaturesFromTransaction] No signature removed because the signatures are in the witness");`